### PR TITLE
Webpage accessibility

### PR DESCRIPTION
**Pull request by anyepeter**
The entire webpage was evaluated against accessibilities and the results are included below 👍 

Things checked 🔖 

- [ ] Page title: The page tile is named portfolio home page

- [ ] Image alternative: Added image alternative to the hamburger icon to indicate its menu and the social media icons.

- [ ] Heading: no issues found.

- [ ] Color: No issues found.

- [ ] Resize: No issue found.

- [ ] Interaction: included the label tag to the input elements in the form.

- [ ] Moving content: No issues found.

- [ ] Multimedia: No issues found.

- [ ] Basic structure: No issues found.
💯

### CODE REVIEW

**Review by awais-amjed — COMMENTED** [review]
Hi @anyepeter , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 

## Invalid Code Review Request

You have submitted a group project as a solo one, we are expecting contributions from all members in collaborative projects. (If you are having issues with a partner, kindly use the `collaboration help` forms from the dashboard.)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -1,168 +1,174 @@
...
-<link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300;900&family=Poppins:wght@200;300&family=Roboto:ital,wght@0,300;1,500&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@300;900&family=Poppins:wght@200;300&family=Roboto:ital,wght@0,300;1,500&display=swap"
+        rel="stylesheet">
 
     <link rel="stylesheet" href="style.css">
 
-    <title>Document</title>
+    <title>Portfolio HomePage</title>
```
Comment (prolajumokeoni) [inline index.html]: - [ ] Kindly add a better portfolio name like `Anye's portfolio` this would be a better name 👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @anyepeter and @michaelasong, 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good PR title and Description



## Required Changes ♻️

- [ ] Kindly check the color contrast report [here](https://www.aditus.io/button-contrast-checker/rawcdn-githack-com-2022-08-31-at-20-18-00-049?failing) some buttons are failing, please fix this 👍🏿 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions
-N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**